### PR TITLE
Update Set-NetTCPSetting.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated description for AutoReusePortRangeStartPort

fixes #https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2982

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

CC: @scanum

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2012r2-ps/nettcpip/Set-NetTCPSetting.md [hunk 1]
```diff
@@ -93,9 +93,9 @@ Accept wildcard characters: False
 ```
 
 ### -AutoReusePortRangeStartPort
-Specifies the number of ports for the auto-reuse port range, which is a port range used for local ephemeral port selection by outbound TCP connections for which either SO_REUSE_UNICASTPORT has been set on the socket, or for which connect() has been called without calling bind() on the socket.
+Specifies the starting port for the auto-reuse port range.
```
Comment (dariomws) [inline docset/winserver2012r2-ps/nettcpip/Set-NetTCPSetting.md:96]:
```suggestion
```

File: docset/winserver2012r2-ps/nettcpip/Set-NetTCPSetting.md [hunk 2]
```diff
@@ -93,9 +93,9 @@ Accept wildcard characters: False
...
 
-If you specify 0, the auto-reuse feature is disabled and ephemeral ports are drawn instead from the dynamic port range as specified by *DynamicPortRangeStartPort* and *DynamicPortRangeNumberOfPorts*, even if SO_REUSE_UNICASTPORT is set on a socket.
+This parameter sets the starting port to send and receive TCP traffic, which is a port range used for local ephemeral port selection by outbound TCP connections for which either SO_REUSE_UNICASTPORT has been set on the socket, or for which connect() has been called without calling bind() on the socket.
```
Comment (dariomws) [inline docset/winserver2012r2-ps/nettcpip/Set-NetTCPSetting.md:98]:
```suggestion
Specifies the starting port for the auto-reuse port range, which is a port range used for local ephemeral port selection by outbound TCP connections for which either SO_REUSE_UNICASTPORT has been set on the socket, or for which connect() has been called without calling bind() on the socket.

This parameter is required if you specify a value other than 0 for the AutoReusePortRangeStartPort parameter.
```

**Review by dariomws — APPROVED** [review]
It looks good, but I suggest some improvements.

**Review by get-itips — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Comment by FlorentATo** [comment]
Great explanation, thank you @msbemba 👍

**Comment by scanum** [comment]
@tiburd could you please help with this one? Thanks!

**Pull request merged**